### PR TITLE
Feature: Show the "send to" option in the main menu

### LINKED ISSUE #11283: Feature: Show "send to" in the main context menu
### What feature or improvement do you think would benefit Files?

We currently load the send to options in the "show more options" menu, these options are commonly accessed so we should change the behavior and show them in the main menu. We do something similar for the "open with" menu.

### Requirements

- Show send to menu item in the main context menu

### Files Version

v2.4.34

### Windows Version

Windows 11

### Comments

_No response_

### PR DESCRIPTION
**Pull request by hecksmosis**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #11283

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

### CODE REVIEW

**Comment by yaira2** [comment]
Looks like I need to add an even larger delay between tests in the CI.

**Comment by yaira2** [comment]
Can you apply this to the recent files widget?

File: src/Files.App/Strings/en-US/Resources.resw [hunk 1]
```diff
@@ -513,6 +513,9 @@
   <data name="BaseLayoutItemContextFlyoutOpenItemWith.Text" xml:space="preserve">
     <value>Open With</value>
   </data>
+  <data name="BaseLayoutItemContextFlyoutSendTo.Text" xml:space="preserve">
```
Comment (yaira2) [inline src/Files.App/Strings/en-US/Resources.resw:516]:
```suggestion
  <data name="SendTo" xml:space="preserve">
```

**Comment by hecksmosis** [comment]
@yair100 Added it to recent items

**Comment by yaira2** [comment]
`uE17D` looked a little better imo

File: src/Files.App/Strings/en-US/Resources.resw [hunk 2]
```diff
@@ -513,6 +513,9 @@
   <data name="BaseLayoutItemContextFlyoutOpenItemWith.Text" xml:space="preserve">
     <value>Open With</value>
   </data>
+  <data name="SendTo" xml:space="preserve">
+    <value>Send To</value>
```
Comment (yaira2) [inline src/Files.App/Strings/en-US/Resources.resw:517]:
```suggestion
    <value>Send to</value>
```

**Comment by hecksmosis** [comment]
It looked a bit off to have both icons being the same in Recent Files

**Comment by yaira2** [comment]
Isn't it a little different?

**Comment by hecksmosis** [comment]
It doesn't have the color, the glyph is exactly the same as the one used for "Open with" while it's loading

**Comment by yaira2** [comment]
Let's leave the icon blank for now until we can create our own.

**Comment by hecksmosis** [comment]
Ok

**Comment by hecksmosis** [comment]
Done

**Review by yair100 — APPROVED** [review]
LGTM

**Pull request merged**